app/controllers/users_controller.rb

```diff
@@ -50,7 +55,7 @@ def admin
   def assign_admin
     if current_user.admin
       user = User.find params[:id]
-      good_params = params.permit(:publisher, :editor, :admin).select{|k,v| v.present?}
+      good_params = params.permit(:librarian, :editor, :admin).select{|k,v| v.present?}
```

Comment:
Unused block argument - `k`. If it's necessary, use `_` or `_k` as an argument name to indicate that it won't be used.<br>Line is too long. [87/80]<br>Space missing after comma.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.